stacks/patterns/index.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Title: Stacks
+ * Slug: stacks/index
```

Comment:
Should this be called `stacks/home` and be named `home.php` instead, since it's used in the home template?